broker/src/main/java/io/camunda/zeebe/broker/Broker.java

```diff
@@ -136,6 +138,8 @@
   private final List<ZeebePartition> partitions = new ArrayList<>();
   private BrokerAdminService brokerAdminService;
 
+  private final TestCompanionClass testCompanionObject = new TestCompanionClass();
```

Comment:
:clown_face: Not a fan of test specific code, but I also don't have a good suggestion for now. If we could use simply the clusterServices instead, that might help, but that's also not the best.